db/seeds.rb

```diff
@@ -45,6 +45,10 @@
 
 [fund1, fund2].each do |fund|
   ActsAsTenant.with_tenant(fund) do
+    lines = ['Main', 'Spanish'].map do |line|
```

Comment:
this does change the seed lines to main/spanish - can do dc/md/va again if we like.

Reply:
done! Kept with the cat themes for the second fund's lines.